.github/workflows/update-owasp-db.yaml

```diff
@@ -0,0 +1,20 @@
+name: Update Vulnerability Database
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '46 20 * * 2' # use a somewhat random time to avoid producing load spikes on the GH actions infrastructure
```

Comment:
```suggestion
    - cron: '46 20 * * 2' # At 08:46 PM, only on Tuesday, random time to avoid producing load spikes on the GH actions infrastructure
```